packages/flutter_tools/lib/src/desktop.dart

```diff
@@ -3,11 +3,12 @@
 // found in the LICENSE file.
 
 import 'base/platform.dart';
+import 'version.dart';
 
 // Only launch or display desktop embedding devices if
 // `FLUTTER_DESKTOP_EMBEDDING` environment variable is set to true.
 bool get flutterDesktopEnabled {
   _flutterDesktopEnabled ??= platform.environment['FLUTTER_DESKTOP_EMBEDDING']?.toLowerCase() == 'true';
-  return _flutterDesktopEnabled;
+  return _flutterDesktopEnabled && FlutterVersion.instance.getBranchName() != 'stable';
```

Comment:
Can 'stable' be a constant or an enum member? Or maybe instead:

`FlutterVersion.instance.isStable`

Reply:
Done